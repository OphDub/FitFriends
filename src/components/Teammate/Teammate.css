.Teammate {
  border: 1px solid black;
  padding: 5px; }

.teammate-name,
.teammate-tally,
.teammate-posted,
h6, p {
  margin: 0px; }

.Teammate ul {
  width: 50%; }

.Teammate li {
  list-style: none;
  display: flex;
  flex-direction: row;
  align-items: center;
  justify-content: space-between; }
