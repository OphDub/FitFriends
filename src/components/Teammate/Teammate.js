import React from 'react';
import './Teammate.css';

export const Teammate = ({ name, workoutTally, workoutsPosted }) => {
  return(
    <article className="Teammate">
      <h4 className="teammate-name">{name}</h4>
      <ul>
        <li className="teammate-tally">
          <h6>Workout Tally:</h6>
          <p>{workoutTally}</p>
        </li>
        <li className="teammate-posted">
          <h6>Workouts Posted:</h6>
          <p>{workoutsPosted}</p>
        </li>
      </ul>
    </article>
  )
}