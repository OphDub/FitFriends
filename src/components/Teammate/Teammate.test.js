/* eslint-disable */
import React from 'react';
import { shallow } from 'enzyme';
import { Teammate } from './Teammate';

describe('TEAMMATE', () => {
  it('should match snapshot', () => {
    const renderedComponent = shallow(<Teammate/>);

    expect(renderedComponent).toMatchSnapshot();
  });
});