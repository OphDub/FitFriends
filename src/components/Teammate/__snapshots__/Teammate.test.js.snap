// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TEAMMATE should match snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h4
          className="teammate-name"
/>,
        <ul>
          <li
                    className="teammate-tally"
          >
                    <h6>
                              Workout Tally:
                    </h6>
                    <p />
          </li>
          <li
                    className="teammate-posted"
          >
                    <h6>
                              Workouts Posted:
                    </h6>
                    <p />
          </li>
</ul>,
      ],
      "className": "Teammate",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": undefined,
          "className": "teammate-name",
        },
        "ref": null,
        "rendered": null,
        "type": "h4",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <li
              className="teammate-tally"
>
              <h6>
                            Workout Tally:
              </h6>
              <p />
</li>,
            <li
              className="teammate-posted"
>
              <h6>
                            Workouts Posted:
              </h6>
              <p />
</li>,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h6>
                  Workout Tally:
</h6>,
                <p />,
              ],
              "className": "teammate-tally",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Workout Tally:",
                },
                "ref": null,
                "rendered": "Workout Tally:",
                "type": "h6",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": undefined,
                },
                "ref": null,
                "rendered": null,
                "type": "p",
              },
            ],
            "type": "li",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h6>
                  Workouts Posted:
</h6>,
                <p />,
              ],
              "className": "teammate-posted",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Workouts Posted:",
                },
                "ref": null,
                "rendered": "Workouts Posted:",
                "type": "h6",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": undefined,
                },
                "ref": null,
                "rendered": null,
                "type": "p",
              },
            ],
            "type": "li",
          },
        ],
        "type": "ul",
      },
    ],
    "type": "article",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h4
            className="teammate-name"
/>,
          <ul>
            <li
                        className="teammate-tally"
            >
                        <h6>
                                    Workout Tally:
                        </h6>
                        <p />
            </li>
            <li
                        className="teammate-posted"
            >
                        <h6>
                                    Workouts Posted:
                        </h6>
                        <p />
            </li>
</ul>,
        ],
        "className": "Teammate",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": undefined,
            "className": "teammate-name",
          },
          "ref": null,
          "rendered": null,
          "type": "h4",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <li
                className="teammate-tally"
>
                <h6>
                                Workout Tally:
                </h6>
                <p />
</li>,
              <li
                className="teammate-posted"
>
                <h6>
                                Workouts Posted:
                </h6>
                <p />
</li>,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h6>
                    Workout Tally:
</h6>,
                  <p />,
                ],
                "className": "teammate-tally",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Workout Tally:",
                  },
                  "ref": null,
                  "rendered": "Workout Tally:",
                  "type": "h6",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": undefined,
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "p",
                },
              ],
              "type": "li",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h6>
                    Workouts Posted:
</h6>,
                  <p />,
                ],
                "className": "teammate-posted",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Workouts Posted:",
                  },
                  "ref": null,
                  "rendered": "Workouts Posted:",
                  "type": "h6",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": undefined,
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "p",
                },
              ],
              "type": "li",
            },
          ],
          "type": "ul",
        },
      ],
      "type": "article",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
